src/software/logger/coloured_cout_sink.h

```diff
@@ -35,6 +35,9 @@ class ColouredCoutSink
      */
     void displayColouredLog(g3::LogMessageMover log_entry);
 
+    std::chrono::_V2::system_clock::duration LOG_INTERVAL_TIMESTAMP =
+        std::chrono::nanoseconds(2 * 1000000000);
```

Comment:
```suggestion
        std::chrono::seconds(2);
```
I believe this results in the same duration